api/v1/app.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/python3
+
+"""
+Flask web application api
+"""
+
+from flask import Flask
+from models import storage
+from api.v1.views import app_views
+import os
+
+
+app = Flask(__name__)
+app.register_blueprint(app_views, url_prefix='/api/v1')
+app.config['JSONIFY_PRETTYPRINT_REGULAR'] = True
+
+
+@app.teardown_appcontext
+def teardown_db(exception):
+    """closes the storage on teardown"""
+    storage.close()
+
+@app.route('/hello') # example.com/hello/Anthony
+def hello():
+    return 'Hello Crisoy!' # returns hello Anthony!
+
+if __name__ == '__main__':
+    my_host = os.getenv('HBNB_API_HOST')
+    my_port = os.getenv('HBNB_API_PORT')
+    app.run(host=my_host, port=my_port, threaded=True)
```

Comment:
Need to convert ENV variable HBNB_API_PORT to integer (just cast it in the app.run call i.e port=int(my_port)

Reply:
Done